build_runner/lib/src/generate/build_definition.dart

```diff
@@ -101,8 +101,13 @@ class _Loader {
       Set<AssetId> conflictingOutputs;
 
       await logTimedAsync(_logger, 'Building new asset graph', () async {
-        assetGraph = await AssetGraph.build(_buildPhases, inputSources,
-            internalSources, _options.packageGraph, _environment.reader);
+        try {
+          assetGraph = await AssetGraph.build(_buildPhases, inputSources,
+              internalSources, _options.packageGraph, _environment.reader);
+        } on DuplicateAssetNodeException catch (e) {
+          _logger.severe('$e');
```

Comment:
lets attach the stack trace as well, so we get it in verbose mode.

Reply:
Done